packages/okta-react-native/README.md

```diff
@@ -50,145 +40,392 @@ After you have created the application there are two more values you will need t
 
 These values will be used in your React application to setup the OpenID Connect flow with Okta.
 
-## Installation
+## Getting started
 
 This library is available through [npm](https://www.npmjs.com/package/@okta/okta-react-native). To install it, simply add it to your project:
 
-```bash
-# npm
-npm install --save @okta/okta-react-native
+```
+$ npm install @okta/okta-react-native --save
+$ react-native link @okta/okta-react-native
+```
+
+Perform the following Manual installation steps if you're not using `react-native link`.
+
+#### Manual installation (Optional)
+
+##### iOS
+
+1. In XCode, in the project navigator, right click `Libraries` ➜ `Add Files to [your project's name]`
+2. Go to `node_modules` ➜ `@okta/okta-react-native` and add `ReactNativeOktaSdkBridge.xcodeproj`
+3. In XCode, in the project navigator, select your project. Add `libReactNativeOktaSdkBridge.a` to your project's `Build Phases` ➜ `Link Binary With Libraries`
+4. Run your project (`Cmd+R`)<
+
+##### Android
+
+1. Open up `android/app/src/main/java/[...]/MainApplication.java`
+  - Add `import com.oktareactnative.OktaSdkBridgePackage;` to the imports at the top of the file
+  - Add `new OktaSdkBridgePackage()` to the list returned by the `getPackages()` method
+2. Append the following lines to `android/settings.gradle`:
+    ```
+    include ':@okta/okta-react-native'
+    project(':@okta/okta-react-native').projectDir = new File(rootProject.projectDir, '../node_modules/@okta/okta-react-native/android')
+    ```
+3. Insert the following lines inside the dependencies block in `android/app/build.gradle`:
+    ```
+      compile project(':@okta_okta-react-native')
+    ```
+
+### iOS Setup
+
+To setup iOS, there are three steps that you must take.
+
+1. [Make sure your iOS app's deployment target is `11.0` and above.](#set-ios-deployment-target)
+2. [Install Okta Open ID Connect iOS.](#install-okta-open-id-connect-ios)
+3. [Make sure you also configure Swift.](#swift-configuration)
+
+#### Set iOS Deployment Target
+
+This library supports iOS version `11.0` and above. Go to your project -> `Build settings` -> `iOS Deployment Target`, and set it to at least version `11.0`.  
+
+#### Install Okta Open ID Connect iOS
+This library depends on the native [Okta OIDC iOS](https://github.com/okta/okta-oidc-ios) library. This library is not distributed as part of the React Native library to keep your dependency management consistent. 
+
+There are three ways to add Okta OIDC iOS to your dependencies:
+
+1. **CocoaPods**
+   With [CocoaPods](https://guides.cocoapods.org/using/getting-started.html), add the following line to
+   your `Podfile`:
+
+    ```   
+    pod 'OktaOidc', '~> 3.0'
```

Comment:
I would show the whole podfile example. `use_frameworks` is also critical command in podfile
So, something like that:

`platform :ios, '11.0'

use_frameworks!

target 'YourTarget' do
pod 'OktaOidc', '~> 3.0'
end`